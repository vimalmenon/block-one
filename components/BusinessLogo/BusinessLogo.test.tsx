import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

import { BusinessLogo } from './'

describe('BusinessLogo Component', () => {
  test('BusinessLogo: Snapshot testing', () => {
    const wrapper = shallow(<BusinessLogo />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
