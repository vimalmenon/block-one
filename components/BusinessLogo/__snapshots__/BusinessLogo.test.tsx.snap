// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BusinessLogo Component BusinessLogo: Snapshot testing 1`] = `
<div
  className="flex w-48 flex-col"
>
  <div
    className="text-5xl self-center font-bold text-blue-700"
  >
    FB
  </div>
  <div
    className="text-sm self-center text-yellow-500"
  >
    Fictitious Bank
  </div>
</div>
`;
